packages/openapi-v3/src/__tests__/unit/enhancers/fixtures/security.spec.extension.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {bind} from '@loopback/core';
+import debugModule from 'debug';
+import {inspect} from 'util';
+import {
+  mergeOpenAPISpec,
+  ReferenceObject,
+  SecuritySchemeObject,
+} from '../../../..';
+import {asSpecEnhancer, OASEnhancer} from '../../../../enhancers/types';
+import {OpenApiSpec} from '../../../../types';
+const debug = debugModule('loopback:openapi:spec-enhancer');
+
+export type SecuritySchemeObjects = {
+  [securityScheme: string]: SecuritySchemeObject | ReferenceObject;
+};
+
+export const SECURITY_SCHEME_SPEC: SecuritySchemeObjects = {
+  bearerAuth: {
```

Comment:
@emonddr Nitpick: Would it make sense to use `jwt` specifically rather than `bearerAuth` to be more descriptive?
````ts
      "jwt": {
        "type": "http",
        "scheme": "bearer",
        "bearerFormat": "jwt"
      }
````
It would make the openapi security object more informative IMO as JWT's are nearly always used as bearer tokens where are the former can vary.
````
  "security": [
    {
      "jwt": []
    }
  ],
````